rpc/src/client.rs

```diff
@@ -150,5 +153,80 @@ mod native {
 
 #[cfg(target_arch = "wasm32")]
 mod wasm {
+    use std::fmt;
+
+    use async_trait::async_trait;
+    use jsonrpsee::core::client::{BatchResponse, ClientT, Subscription, SubscriptionClientT};
+    use jsonrpsee::core::params::BatchRequestBuilder;
+    use jsonrpsee::core::traits::ToRpcParams;
+    use jsonrpsee::core::Error as JrpcError;
+    use serde::de::DeserializeOwned;
+
     // TODO: implement HttpClient with `fetch`
+    pub struct Client;
+
+    impl Client {
+        pub async fn new(_conn_str: &str, _auth_token: Option<&str>) -> crate::Result<Self> {
+            // TODO: jsonrpsee_wasm_client doesn't support setting headers
```

Comment:
wasm-client and more specifically websockets are not an option for us as due to this reason. This should be done with `fetch` and that's why we didn't implement it yet :) Can we just revert changes on the `toml` and `src` there and just properly guard tests for now?

Reply:
Ah, I thought we'd be able to achieve single interface with just jsonrpsee - removed the stubs for wasm here and added `#!cfg(..)` to rpc tests so that we can run clippy.